src/main/kotlin/no/nav/familie/klage/behandling/BehandlingService.kt

```diff
@@ -54,6 +56,10 @@ class BehandlingService(
         ).id
     }
 
+    fun hentBehandlinger(eksternId: String, fagsystem: Fagsystem): List<Klagebehandling> {
```

Comment:
eksternFagsakId. Burde kanskje hete finnBehandlinger for å følge patternet med hvis det returneres en liste så pleier vi å prefixe med `finn..`, mens hvis det returneres kun en så prefixes det med `hent..`